lib/plugsnag.ex

```diff
@@ -4,18 +4,25 @@ defmodule Plugsnag do
       use Plug.ErrorHandler
       import Plugsnag
 
-      # We don't want to bugsnag for errors which have plug_status and plug_status is valid < 500
-      defp handle_errors(_conn, %{reason: %{plug_status: plug_status}}) when plug_status < 500 do
-        nil
-      end
-
-      if :code.is_loaded(Ecto) do
-        defp handle_errors(conn, %{reason: %Ecto.NoResultsError{}}) do
+      defp handle_errors(conn, %{reason: exception, kind: :error} = assigns) do
```

Comment:
`kind` can also be `:throw` or `:exit`, so we need to match on `:error` here, as `Plug.Exception.status/1` has a fallback case that returns `500`, even for non-exception types.